canonical-web-and-design/ubuntu.com
# Remove non-pdf robotics whitepaper

## Done

- Removed the non-pdf test of the robotics whitepaper and pointed to the traditional /engage page version (fyi @anthonydillon)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/whitepapers/beta/robotics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to the engage page version
- See that the entire /whitepaper directory has been removed

## Issue / Card

Fixes #9457
